apps/admin-gui/src/app/shared/components/dialogs/remove-service-from-resource-dialog/remove-service-from-resource-dialog.component.html

```diff
@@ -0,0 +1,38 @@
+<h1 mat-dialog-title>{{'DIALOGS.REMOVE_SERVICE_FROM_RESOURCE.TITLE' | translate}}</h1>
+<div mat-dialog-content>
+  <p>
+    {{'DIALOGS.REMOVE_SERVICE_FROM_RESOURCE.DESCRIPTION' | translate}}
+  </p>
+
+  <div class="font-weight-bold">
+    {{'DIALOGS.REMOVE_SERVICE_FROM_RESOURCE.ASK' | translate}}
+  </div>
+  <mat-spinner *ngIf="loading" class="mr-auto ml-auto"></mat-spinner>
```

Comment:
also the spinner needs theme so it have proper color. Look to other dialogs how they set theme to whole dialog